frontend/src/container/ExplorerOptions/ExplorerOptions.styles.scss

```diff
@@ -41,6 +41,11 @@
 
 				font-weight: 500;
 			}
+
+			&.ant-btn-round:disabled {
+				background-color: rgba(209, 209, 209, 0.074);
+				color: #5f5f5f;
```

Comment:
Use variables, Handle light mode

Reply:
This looks good for both dark and light mode.